lib/publisher/apppublisher.go

```diff
@@ -10,9 +10,12 @@ const (
 	Database   RemoteAppType = "REMOTE_DATABASE"
 	Desktop    RemoteAppType = "REMOTE_DESKTOP"
 	Kubernetes RemoteAppType = "REMOTE_KUBERNETES"
+	Invalid    RemoteAppType = "INVALID"
 
 	//default sqs queue name
 	defaultSQSQueueName = "remote-access-resource-change-notification-queue"
+
+	defaultDelayInSeconds = 30
```

Comment:
let's add a comment on why we need to add this delay. It will be easier to refer back to it.